src/Components/CreateImageWizard/steps/TargetEnvironment/Aws/index.tsx

```diff
@@ -12,6 +12,10 @@ import {
   HelperText,
   HelperTextItem,
   Button,
+  Select,
```

Comment:
For this to work we would need to update to include the `region` which we obviously can't do since we'd then need workers in each region.
https://github.com/osbuild/image-builder-crc/blob/48d1c3b953b106fd53bf98ff32fbfc7719206456/internal/v1/api.yaml#L1316-L1330

Seems like a hard-blocker towards using a dropdown for the upload region